recipes/anacal/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "AnaCal" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mr-superonion/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0ebf6f23306d5049d1659c7ec6ee5f85dee5e036ce74542da88b6270d545752f
+
+
+build:
+  number: 0
+  skip: True  # [py<310 or win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+  host:
+    - python
+    - setuptools >=38
+    - fftw >=3.3
+    - pybind11 >=2.2
```

```suggestion
    - fftw >=3.3
    - pybind11 >=2.2
    - fftw
    - pybind11
```

You need entries without the pins to ensure you get the global pins from conda-forge properly.